app/views/comments/_form.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="comment-form-wrapper" style="background-color:#f8f8f8; border: 1px solid #e7e7e7;padding: 18px;">
-<form class="comment-form" id="comment-form" data-remote="true" class="well" <% if !local_assigns[:is_answer].blank? %> action= "/answers/create/<%= @node.nid %>" <% elsif !local_assigns[:aid].blank? %> action= "/comment/answer_create/<%= aid %>" <% else %> action="/comment/create/<%= @node.nid %><%= "?type=question" if local_assigns[:type]=="question" %>" <% end %> <% if local_assigns[:aid].blank? %>method="post"<% end %>>
+<form class="comment-form" id="comment-form-<%= Time.now.to_s.gsub(" ", "") %><%= rand(10000) %>" data-remote="true" class="well" <% if !local_assigns[:is_answer].blank? %> action= "/answers/create/<%= @node.nid %>" <% elsif !local_assigns[:aid].blank? %> action= "/comment/answer_create/<%= aid %>" <% else %> action="/comment/create/<%= @node.nid %><%= "?type=question" if local_assigns[:type]=="question" %>" <% end %> <% if local_assigns[:aid].blank? %>method="post"<% end %>>
```

Comment:
Hi @CleverFool77 !!! This is great, you have just the right idea. However, I think two possibilities may be good; one, to not change the id, but to use a new class (in case some other function depends on the old format of id), AND to consider using the comment `cid` if it exists. Here, are we able to use `aid`? @ViditChitkara what do you think?

Thank you!!!

Reply:
Yes, I think we need to check if some other js function(drag drop or editor) depends on the comment IDs.

Reply:
I think the cid might not exist for a new comment form. Maybe that's the reason why cid has not been considered? @CleverFool77

Reply:
Also, the tests for the reply to comment features are not yet written so we may need to manually test if the reply to comment feature is not affected, image upload is working, propose title, etc. which appear at the bottom of the comment textbox are working.
Will be writing the tests asap. @CleverFool77 if you want, we'd love your help in writing tests.

Reply:
Hi @jywarren I went through all the js functions for comments. And I found that there is no use of comment-form. Instead mostly places in js have used classes.
Regarding other ids, I will look into it.
And as @ViditChitkara mentioned. Earlier, I was considering to use cid first but then as I went though the code base I realised the comment-form doesn't have cid. And That's why I went for current method.
And Sure, It would be great to help in writing test for comment section.